File: java/javadpbo/src/com/sem4dpbo/week4/contohsoal1/Matriks.java
package com.sem4dpbo.week4.contohsoal1;

public class Matriks {
    private int baris;
    private int kolom;
    private int[][] mat;

    Matriks(){}

    Matriks(int baris, int kolom){
        this.baris = baris;
        this.kolom = kolom;
        mat = new int[baris][kolom];
    }

//    get and set baris
    public void setBaris(int baris) {
        this.baris = baris;
    }

    public int getBaris() {
        return baris;
    }

//    get and set kolom
    public void setKolom(int kolom) {
        this.kolom = kolom;
    }

    public int getKolom() {
        return kolom;
    }

//    get and set matriks
    public void setMat() {
        mat = new int[baris][kolom];
    }

    public int[][] getMat() {
        return mat;
    }

//    get and set sel
    public void setSel(int baris, int kolom, int nilai){
        mat[baris][kolom] = nilai;
    }

    public int getSel(int baris, int kolom){
        return mat[baris][kolom];
    }

}

File: php/4th_week/contohsoal2/Matriks.php
<?php

class Matriks {
    private $baris;
    private $kolom;
    private $mat;

    function __construct()
    {   
    }


//    get and set baris
    function setBaris($baris) {
        $this->baris = $baris;
    }

    function getBaris() {
        return $this->baris;
    }

//    get and set kolom
    function setKolom($kolom) {
        $this->kolom = $kolom;
    }

    function getKolom() {
        return $this->kolom;
    }

//    get and set matriks
    function setMat($mat) {
        $this->mat = $mat;
    }

    function getMat() {
        return $this->mat;
    }

//    get and set sel
    function setSel($baris, $kolom, $nilai){
        $this->mat[$baris][$kolom] = $nilai;
    }

    function getSel($baris, $kolom){
        return $this->mat[$baris][$kolom];
    }
}

?>